girderformindlogger/models/user.py

```diff
@@ -427,8 +436,16 @@ def createUser(self, login, password, displayName="", email="",
         requireApproval = Setting(
         ).get(SettingKey.REGISTRATION_POLICY) == 'approve'
         email = "" if not email else email
+
+        login = login.lower().strip()
```

Comment:
I noticed that there are several plugins and platforms that using our backend code.

they all use createUser function in models/user.py, but I don't know we have to remove login field there as well because they might be involved in other project.

Reply:
@jj105 Is there any other plugins/platforms use our backend? not only our web/admin/app?

Reply:
at least, they are not able to call a function([createUser](https://github.com/ChildMindInstitute/mindlogger-app-backend/blob/5d8401ebc3e497a9234bb908de5c533a09a97939/girderformindlogger/api/v1/user.py#L806-L814)) of our user model?

Reply:
they call createUser function of our user model.

Reply:
simply removing login field in createUser function can cause issue on them.

Reply:
oh that's good to know! can you refer to where it's being used in other place?

Reply:
just search createUser text in the entire code.
you will be able to see lots of places that using it.

Reply:
let's skip this for now then. but let's keep in mind this part should be updated later